cryptographic_estimators/MQEstimator/degree_of_regularity.py

```diff
@@ -87,40 +77,32 @@ def regular_system(n: int, degrees: list[int]):
 
 
 def semi_regular_system(n: int, degrees: list[int], q=None):
-    r"""
-    Return the degree of regularity for semi-regular system
+    """Return the degree of regularity for semi-regular system.
 
-    The degree of regularity of a semi-regular system $(f_1, \ldots, f_m)$ of respective degrees $d_1, \ldots, d_m$ is
-    given by the index of the first non-positive coefficient of
+    The degree of regularity of a semi-regular system (f_1, ..., f_m) of respective degrees d_1, ..., d_m is
+    given by the index of the first non-positive coefficient of:
 
-    .. MATH::
+    (prod_{i=1}^{m} (1 - z^{d_i})) / (1 - z)^{n}
```

Comment:
will this be rendered as math formula in the documentation?

Reply:
I am not sure, and I would prefer that this was not a reason to block the PRs since the aesthetics of the documentation was not given to me as a requirement for the migration.

Instead, I will create an issue where we can track any concerns in this regard, so that I can explore and resolve them when the migration at the functional level is complete.

If you find any other issues in the PRs following this one, feel free to comment and I will add them to the issue created to address them later.

Reply:
The issue is https://github.com/Crypto-TII/CryptographicEstimators/issues/198.  Just to be sure, if you want to create a direct link from your future review comments, you can do it by just writing  the number sign (#) + 198. If you don't remember the number don't worry,  just let me know in the comments and I will create the link/update the issue list!